src/vips/vobject.lua

```diff
@@ -55,12 +54,11 @@ local vobject_mt = {
 
         get = function(self, name)
             log.msg("vobject.get")
-            log.msg("  self =", self)
             log.msg("  name =", name)
 
             local gtype = self:get_typeof(name)
             if gtype == 0 then
-                error(verror.get())
+                return false
```

Comment:
But what if you have a bool type? Isn't false going to look like a valid return?

My memory is you use `get_typeof(name)` to test for name exists (0 is not a valid type) and that won't throw, and `get(name)` to actually fetch the metadata, which will throw an error for no such name.

pyvips and ruby-vips work the same way: 0 return for `get_typeof()` for not-defined, exception for `get()` for not-defined.